cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/13-school-results.html

```diff
@@ -0,0 +1,91 @@
+        <p>Ask the school to provide the metrics you care about, such as:</p>
+        <ul>
+            <li>On-time completion rate</li>
+            <li>Job placement rate</li>
+            <li>Loan default rate</li>
+            <li>Loan repayment rate</li>
+            <li>Persistence and retention rate</li>
+        </ul>
+        <p>You should consider asking multiple schools for this information, so you can compare this school’s results.</p>
+    </div>
+
+    <div class="school-results_undergrad-content" data-program-type-content="undergrad">
+        <p>Now that you understand how much debt you’re considering, let’s look at some statistics that indicate how well the school serves students. You’re investing a lot—make sure it pays off.</p>
+
+        <h3 class="header_border-bottom"><span data-school-item="name"></span></h3>
+        <div class="school-results_basics">
+            <div class="school-results_basics__half">
+                <div class="school-fact">
+                    <strong>Location:</strong>
+                    <span data-school-item="location">Lubbock, TX</span>
+                </div>
+                <div>
+                    <strong>Type:</strong>
+                    <span data-school-item="control">Public University</span>
+                </div>
+                <div>
+                    <strong>Student Body:</strong>
+                    <span data-school-item="studentBody">30,330</span>
+                </div>
+            </div>
+            <div class="school-results_basics__half">
+                <div>
+                    <strong>Highest degree offered:</strong>
+                    <span data-school-item="highestDegree">PhD</span>
+                </div>
+                <div>
+                    <strong>Number of Programs:</strong>
+                    <span data-school-item="programCount">53</span>
+                </div>
+                <div>
+                    <strong>Most popular programs:</strong>
+                    <span data-school-item="popularPrograms">Business, Management, Marketing and Related Support Services</span>
+                </div>
+            </div>
+        </div>
+
+        {{ notification.render(
+            'warning',
+            true,
+            '',
+            'This school is under Heightened Cash Monitoring by the U.S. Department of Education. <a href="">Find out more.</a>' | safe
+        ) }}
+
+        <h4>Graduation Rate</h4>
+        <div class="school-results_rank-display">
+            <div class="big-percentile"><span data-school-item="grad-rate_control"></span></div>
+            <div><span data-school-item="name"></span>'s graduation rate is <strong><span data-school-item="grad-rate_control"></span></strong>. If you are going to school to earn more money, you need to complete your degree to get that payoff.</div>
```

Comment:
Do you need the span that's wrapped in a strong or can you have `<strong data-school-item="grad-rate_control"></strong>`?

Reply:
I think that may be required for the Javascript? I'll make a note to check and remove the `<span>`s if it isn't.

Reply:
Any `data-school-item` element should work here. It might be better if it was styled in the CSS rather than a `<STRONG>`, whatever is easiest.

Reply:
Thanks, @mistergone! I'll push an update

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/24b0e7e7c8e53c836c938545aa6c87acfafa06a3